games-util/springlobby/springlobby-0.264.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+WX_GTK_VER="3.0"
+inherit cmake-utils eutils flag-o-matic wxwidgets
+
+DESCRIPTION="The official lobby client for SpringRTS community games"
+HOMEPAGE="https://springlobby.info"
+SRC_URI="https://www.springlobby.info/tarballs/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+RESTRICT="mirror"
+IUSE="debug +libnotify +nls +sound"
+
+RDEPEND="
+	>=dev-libs/boost-1.35:=
+	dev-libs/openssl:*
```

Comment:
`:*` here is wrong as not every slot is acceptable.

Reply:
Your right, from ldd output:
libssl.so.1.0.0 => /usr/lib64/libssl.so.1.0.0
This will break if changing slot at runtime.
It should still work with both slots of openssl